Makefile

```diff
@@ -55,7 +55,7 @@ verify-go-mod: go-mod
 
 .PHONY: test-unit
 test-unit: $(BUILD_PATH)
-	$(GO) test -v -test.coverprofile=$(BUILD_PATH)/coverage.out ./...
+	$(GO) test -v -test.coverprofile=$(BUILD_PATH)/coverage.out ./internal/...
```

Comment:
Do we need to restrict test coverage to “internal”?

Reply:
Yeah otherwise we would also execute the e2e tests in `./test` 🤭

Reply:
Ahh I see! I have solved this before with build tags on integration test files (https://mickey.dev/posts/go-build-tags-testing/), but this seems like an equally valid approach :)

Reply:
Hm, let me revert that change, I like the build tags approach more and will follow-up in another PR. :+1: